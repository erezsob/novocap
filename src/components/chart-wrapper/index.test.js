import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import ChartWrapper from './index';

describe('<ChartWrapper />', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(
        <ChartWrapper>
          <dummy-child />
        </ChartWrapper>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders its children', () => {
    const wrapper = shallow(
      <ChartWrapper>
        <dummy />
        <dummy />
      </ChartWrapper>
    );
    expect(wrapper.children()).toHaveLength(2);
  });
});
