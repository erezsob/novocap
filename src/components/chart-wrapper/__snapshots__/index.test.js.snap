// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChartWrapper /> renders correctly 1`] = `
<div
  className="container"
>
  <dummy-child />
</div>
`;
